clustered/integration-test/src/test/java/org/ehcache/clustered/replication/BasicClusteredCacheOpsReplicationTest.java

```diff
@@ -190,6 +191,7 @@ public void testCAS() throws Exception {
   }
 
   @Test
+  @Ignore
```

Comment:
Is this being ignored with expectation for a future repair?  If so, an issue number should be mentioned.